README.md

```diff
@@ -170,6 +171,24 @@ This optional generator, not included with the default generator, creates the ap
 
 The last group of generators are for your convenience if you want to start overriding code included as part of the Rails engine. For example by default the engine provides a simple SessionController, if you run the `rails generate shopify_app:controllers` generator then this code gets copied out into your app so you can start adding to it. Routes and views follow the exact same pattern.
 
+Deploying to a Subdirectory
+---------------------------
```

Comment:
can we call this section `Mounting the Engine` or something. I don't think deploying is the right word to use here